### ISSUE TITLE
[Enhancement] Add the server start/stop to the main page

### ISSUE THREAD

**Issue by giventofly** [dave-lang/webmin-docker#19]
Can it be added on the main web page?

![image](https://github.com/user-attachments/assets/c048ddec-6c8c-4a36-a7b2-8d590964bb9d)

So we can easily restart/stop/start and check the status?

**Comment by dave-lang** [comment]
Which module is that? It looks like 'Servers and Server Status', but that doesn't support restart/stop/start

'Servers and Server Status' provides a simple hook to add more monitor types:
https://github.com/webmin/webmin/blob/46af4889942018832f702d8a4aa14a54bfb34bca/status/status-lib.pl#L239-L264. There's no docs on how this works, but https://github.com/webmin/webmin/blob/master/minecraft/status_monitor.pl provides a good example.

I can add 2 monitor types which should provide some support:
- Docker Up - This will check the response from docker info to work around the many ways dockerd can be configured
- Docker Container Up - Check the health of a specific container

Initial support will require the monitor to be added manually

**Comment by giventofly** [comment]
I don't mind adding manually and work as a beta tester for it, just tell me the steps.

and thanks for your time

**Comment by dave-lang** [comment]
To test adding them, install the latest version of this and then go to the 'servers and server status' module under 'tools'. There will be 2 new monitors for docker and docker containers.

**Comment by giventofly** [comment]
yes, they appear there but still won't appear in the server status section. My ideas was to save clicking around and have an easy way to check the running dockers.

Essentially it would be this: 
![image](https://github.com/user-attachments/assets/ed24313e-404e-4f51-ad72-59e015012534)

to appear:

![image](https://github.com/user-attachments/assets/4a145326-60c3-4de0-8a6c-4242f03df5f2)

with the options to stop/start/restart

**Comment by dave-lang** [comment]
Which module is that? Webmin or virtualmin?

**Comment by giventofly** [comment]
It is on the webmin section, so i'm guessing webmin

![imagem](https://github.com/user-attachments/assets/055ca287-096f-470a-a492-18738c0b4a52)

**Comment by dave-lang** [comment]
After a bit of research that's actually part of VirtualMin (https://forum.virtualmin.com/t/missing-server-status-webmin-only/123132/9)

If I get time to setup Virtualmin I'll take a look at how those monitors work

**Comment by giventofly** [comment]
Awesome, it would be a cool improvement
